executor/inspection_result.go

```diff
@@ -482,13 +510,13 @@ func (criticalErrorInspection) inspectError(ctx context.Context, sctx sessionctx
 				result := inspectionResult{
 					tp: rule.tp,
 					// NOTE: all tables which can be inspected here whose first label must be `instance`
```

Comment:
```suggestion
					// NOTE: all tables which can be inspected here whose first label must be `statusAddress`
```

Reply:
The label name of metric is `instance`, but the value is status address.